rmgpy/rmg/main.py

```diff
@@ -761,7 +777,125 @@ def execute(self, **kwargs):
         logging.info('The final model edge has %s species and %s reactions' % (edgeSpec, edgeReac))
         
         self.finish()
-
+    
+    def makeSeedMech(self,firstTime=False):
+        """
+        causes RMG to make a seed mechanism out of the current chem_annotated.inp and species_dictionary.txt
+        this seed mechnaism is outputted in a seed folder within the run directory and automatically
+        added to as the (or replaces the current) 'Seed' thermo and kinetics libraries in database
+        
+        if run with firstTime=True it will change self.name to be unique within the thermo/kinetics libraries
+        by adding integers to the end of the name to prevent overwritting
+        """
+        
+        logging.info('Making seed mechanism...')
+        
+        name = self.name
+        
+        currentDir = os.getcwd()
+        
+        if self.saveSeedToDatabase:
+            if firstTime: #make sure don't overwrite current libraries
+                thermoNames = self.database.thermo.libraries.keys()
+                kineticsNames = self.database.kinetics.libraries.keys()
+                
+                if name in thermoNames or name in kineticsNames: 
+                    q = 1
+                    while name+str(q) in thermoNames or name+str(q) in kineticsNames:
+                        q += 1
+                    name += str(q)
+                    self.name = name
+            
+        if not os.path.lexists(os.path.join(currentDir,'seed')): #if seed directory does not exist make it
```

Comment:
There isn't a real difference, you could use either.  I'll change it to exists to avoid confusion.